src/web/app/components/question-response-panel/question-response-panel.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="card bg-light mt-4" *ngFor="let question of questions">
+  <div id="question-{{ question.feedbackQuestion.questionNumber }}-responses" class="card-body">
+    <tm-question-text-with-info [questionNumber]="question.feedbackQuestion.questionNumber" [questionDetails]="question.feedbackQuestion.questionDetails" class="question-text"></tm-question-text-with-info>
+    <div id="other-responses" *ngIf="canUserSeeResponses(question), else nonViewableResponse">
+      <tm-single-statistics [question]="question.feedbackQuestion.questionDetails"
+                            [responses]="question.allResponses"
+                            [statistics]="question.questionStatistics"
+                            [isStudent]="true"></tm-single-statistics>
+      <div *ngIf="responseHiddenQuestions.includes(question.feedbackQuestion.questionType), else dontShowAdditionalMsg">
```

Comment:
Similarly here, `responseHiddenQuestions` should not be different whether it is help page or actual page. Can just move the implementation entirely here.

Reply:
fixed in 850dcc8